docs/About-the-Whispir-platform-API.md

```diff
@@ -3,6 +3,7 @@
 Built by developers, for developers, Whispir is the place where you can easily build communications into your app – SMS, email, push, voice, chat, web, social networks and web pages.
 
 ## REST architecture
+## FIXX
```

Comment:
Is this new heading intended to be here? Looks like it may have been accidentally added.